alternator/streams.cc

```diff
@@ -999,6 +999,16 @@ future<executor::request_return_type> executor::get_records(client_state& client
             case cdc::operation::insert:
                 rjson::add(record, "eventName", "INSERT");
                 break;
+            case cdc::operation::service_row_delete:
+            case cdc::operation::service_partition_delete:
+            {
+                auto user_identity = rjson::value();
```

Comment:
nitpick: it's hard to guess what type of value the default constructor of rjson::value() gives you. I prefer rjson::empty_object() to make it clearer (it just calls rjson::value(rapidjson::kObjectType)).